src/main/java/fi/csc/avaa/smear/dto/Event.java

```diff
@@ -15,15 +17,15 @@
     public LocalDate periodEnd;
     public LocalDateTime timestamp;
 
-    public static Event from(Row row) {
+    public static Event from(Record record) {
```

Comment:
small note: Record could be replaced with interface wrapping the actual record class